src/runtime/pkg/katautils/config.go

```diff
@@ -722,8 +722,8 @@ func newFirecrackerHypervisorConfig(h hypervisor) (vc.HypervisorConfig, error) {
 		ImagePath:             image,
 		RootfsType:            rootfsType,
 		FirmwarePath:          firmware,
-		KernelParams:          vc.DeserializeParams(vc.KernelParamFields(kernelParams)),
-		NumVCPUs:              h.defaultVCPUs(),
+		KernelParams:          vc.DeserializeParams(strings.Fields(kernelParams)),
```

Comment:
This change is partially reverting 82ff2db460fc1fbc646e1915cd24b12a2da32b1a. I guess this is not intended and should be dropped. Maybe something went wrong when squashing @c3d 's patch since it still had the previous version.

Reply:
It went wrong as part of my rebase. :-)